cli/flox/src/commands/init/mod.rs

```diff
@@ -389,13 +389,15 @@ pub(crate) struct ProvidedPackage {
     pub name: String,
     /// Path to the package in the catalog
     /// Checked to be non-empty
-    pub rel_path: Vec<String>,
+    pub rel_path: String,
     /// Version of the package in the catalog
     /// "N/A" if not found
     ///
     /// Used for display purposes only,
     /// version constraints should be added based on the original query.
     pub display_version: String,
```

Comment:
suggestion (nb): if we have the original version on this anyway, we should rather expose this as a method than a field